src/backend/marsha/core/defaults.py

```diff
@@ -44,8 +45,10 @@
     "scanning",
     "infected",
     "copying",
+    "initializing",
```

Comment:
I think we should call this state `initialized`, because there is no related pending action.
ie: for harvesting, first we have `harvesting` state, and when done, we have `harvested`
this is not the case for initialisation.